pkg/controller/volume/ephemeral/metrics/metrics.go

```diff
@@ -33,7 +33,7 @@ var (
 		&metrics.CounterOpts{
 			Subsystem:      EphemeralVolumeSubsystem,
 			Name:           "create_total",
-			Help:           "Number of PersistenVolumeClaims creation requests",
+			Help:           "Number of PersistentVolumeClaims creation requests",
```

Comment:
Shouldn't this be "...Claim" (singular)?

Reply:
Agree. cc @ebblake

Reply:
Indeed, singular makes sense to me. I'll go ahead and update along those lines.